src/shared/schemas/v10/index.js

```diff
@@ -0,0 +1,60 @@
+import map from 'lodash/map';
+import each from 'lodash/each';
+import merge from 'lodash/merge';
+import v9Schema from '../v9';
+
+const migration = (_, newRealm) => {
+    const walletData = newRealm.objectForPrimaryKey('Wallet', 9);
+    const newWalletSettings = newRealm.objects('WalletSettings');
+    const newNotificationsSettings = newRealm.objects('NotificationsSettings');
+
+    // Bump wallet version.
+    walletData.version = 10;
+
+    each(newWalletSettings, (settings) => {
+        settings.hideEmptyTransactions = true;
+    });
+
+    each(newNotificationsSettings, (settings) => {
+        settings.notifications = {
+            general: true,
+            confirmations: true,
+            messages: false,
+        };
+    });
```

Comment:
Does this overwrite existing settings that users may have set? For instance, if they have disabled notifications entirely